src/wavesurfer.js

```diff
@@ -289,6 +289,17 @@ var WaveSurfer = {
         return this.isMuted;
     },
 
+    /**
+     * Get the list of current set filters as an array.
+     *
+     * Filters must be set with setFilters method first
+     */
+    getFilters: function() {
+        if (typeof this.backend.filters !== 'undefined') {
```

Comment:
@katspaugh it will return undefined without the check since filters is not set in the wavesurfer.backend by default to empty.

![image](https://cloud.githubusercontent.com/assets/5193884/22847926/7dc29dcc-efbe-11e6-94d2-aba015aecc37.png)

Once it's set via `wavesurfer.backend.setFilter(lowpass)`, it contains the filters in .backend.

![image](https://cloud.githubusercontent.com/assets/5193884/22847947/bcc34e68-efbe-11e6-80a0-93d81c976ee4.png)